apps/code-examples/src/app/code-examples/forms/file-attachment/basic/demo.component.html

```diff
@@ -1,17 +1,15 @@
-<form [formGroup]="formGroup" class="sky-padding-even-lg">
+<form [formGroup]="formGroup">
```

Comment:
curious, why remove the spacing? The code example looks kinda ugly without it 😅 I could see the argument for using the page component instead of a random class

Reply:
The spacing classes are dependent on the outer context and I didn't want consumers copy/pasting something from the code example if our guidelines recommended something else. I could be swayed, however.